### DISCUSSION TITLE
Are there plans to support caching?

### DISCUSSION THREAD

**Discussion by watsy0007** [vitalik/django-ninja#200] [Ideas]
django cache ? or @ninja_cache

**Comment by vitalik** [comment]
since django cache_view is not really compatible, I guess there will be at some pint either utils method or docs for making one

**Reply by watsy0007** [reply]
ok. thanks. 
i already use ninja in production. 
cache and throttle is the last two features i need in ninja .

**Reply by dpgraham4401** [reply]
Any update on this?

**Reply by vitalik** [reply]
@dpgraham4401 

Caching (using standard django caching mechanism) - https://django-ninja.dev/whatsnew_v1/?h=cache#decorators

Throttling - https://django-ninja.dev/guides/throttling/
